tsdb/engine/tsm1/compact.go

```diff
@@ -205,12 +207,14 @@ func (c *DefaultPlanner) Plan(lastWrite time.Time) []string {
 		return nil
 	}
 
+	c.lastPlanCompactedFull = false
+
 	return tsmFiles
```

Comment:
yeah, exactly my thought. Then we could run multiple levels at the same time. Or if there are many in the same level we could split it in two and run two compactions at the same time.